blackjack.rb

```diff
@@ -45,17 +50,24 @@ def show_hands
     ace_decision
   end
 
+
   def split_decision(current_hand)
-    if current_hand.cards.collect(&:face).uniq.length == 1
-      response = prompt.yes?("Would you like to split? \n")
-      if response
+    if current_hand.cards[0].face == current_hand.cards[1].face
+      if prompt.yes?("Would you like to split? \n")
         playah.draw_a_hand.cards = [current_hand.cards.shift, deck.draw]
         current_hand.cards << deck.draw
+        # for split to work for multiple splits you basically need to call this function again for each new hand.
+        # I wasn't sure how to plug in the syntax.
+        #maybe this:
+        #playah.hands_array.each{ |hand| split_decision(hand)}
+
```

Comment:
I ended up making a new method:

  def split_again(current_hand)
    playah.hands_array.each do |hand|
      split_decision(hand) if playah.hands_array.length < 4
    end
  end

Then, I called split_again after show_hands in the split_decision method.